operators/backup-operator/DefaultBackupOperator.js

```diff
@@ -30,18 +30,23 @@ class DefaultBackupOperator extends BaseOperator {
       } else if (changeObjectBody.status.state === CONST.APISERVER.RESOURCE_STATE.DELETE) {
         return DefaultBackupOperator._processDelete(changeObjectBody);
       }
-    }).catch(err => {
-      logger.error('Error occurred in processing request by DefaultBackupOperator', err);
-      return eventmesh.apiServerClient.updateResource({
-        resourceGroup: CONST.APISERVER.RESOURCE_GROUPS.BACKUP,
-        resourceType: CONST.APISERVER.RESOURCE_TYPES.DEFAULT_BACKUP,
-        resourceId: changeObjectBody.metadata.name,
-        status: {
-          state: CONST.APISERVER.RESOURCE_STATE.FAILED,
-          error: utils.buildErrorJson(err)
-        }
+    })
+      .catch(err => {
+        logger.error(`Error occurred in processing request ${changeObjectBody.status.state} for guid ${changeObjectBody.metadata.name} by DefaultBackupOperator`, err);
+        return eventmesh.apiServerClient.updateResource({
+          resourceGroup: CONST.APISERVER.RESOURCE_GROUPS.BACKUP,
+          resourceType: CONST.APISERVER.RESOURCE_TYPES.DEFAULT_BACKUP,
+          resourceId: changeObjectBody.metadata.name,
+          status: {
+            state: CONST.APISERVER.RESOURCE_STATE.FAILED,
+            response: {
+              state: ONST.APISERVER.RESOURCE_STATE.FAILED,
```

Comment:
Missing `C` of `CONST`

Reply:
Fixed. Looks like this file doesn't have any UTs. Not showing in coverage also.
Will try to add missing tests